recipes/icu/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+./configure --prefix="$PREFIX" \
+    --disable-samples \
+    --disable-extras \
+    --disable-icuio \
```

Comment:
Sort of think this should kept. Other apps might rely on being able to use this `stdio` wrapper. As for he rest, :+1:.